<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Bibliothèque</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Titre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Auteur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Année</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_titre">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_auteur">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ajouter">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_livres">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>290</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_modifier">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_rechercher">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_supprimer">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>420</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_genre">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_annee">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
